src/components/InputText/InputText.tsx

```diff
@@ -0,0 +1,48 @@
+import { InputText as PrimeInputText } from 'primereact/inputtext';
+import cn from 'classnames';
+import Label from '../Label/Label';
+import Small from '../Small/Small';
+import { ErrorIcon } from '../../icons';
+
+export default function InputText(props) {
+  const { Icon } = props;
+
+  const iconClassName = cn(
```

Comment:
Can you please define this inside theme, and then use it like this
   icon={(props) =>
        Icon && <Icon {...iconProps} className={props.iconProps.className} />
      }